Tests/ExtDataDriverGridComp.F90

```diff
@@ -403,6 +403,11 @@ subroutine initialize_gc(gc, import_state, export_state, clock, rc)
              _VERIFY(status)
           end if
        END DO
+       ! now add PS from Root if available
+       call ESMF_StateGet(cap%exports(cap%root_id), 'PLE', field, rc=status)
```

Comment:
I know this comment applies more generally in MAPL, but doesn't this trigger an entry in the ESMF Log if `PLE` is not found?  I did a quick check of the API and don't see any other way to do this.  If you agree, please open a ticket with ESMF to create some query capability for this.

Reply:
Yes, it would trigger an entry in the log. There is another way to do this. I can query the type of `PLE`. One option they have is NO_TYPE, if not found. I'll change it to that.

Reply:
Good.  And now that you've stated it, I suspect that is precisely how they expect us to handle this situation.